src/imager.jsx

```diff
@@ -79,9 +82,39 @@ module.exports = function (config) {
       }
     },
 
+    getLowSrc: function () {
```

Comment:
This ideally would have been a different PR but let's not make it more complicated :-)